data/json/furniture_and_terrain/furniture-plumbing.json

```diff
@@ -98,7 +102,7 @@
     "type": "furniture",
     "id": "f_water_heater",
     "name": "water heater",
-    "looks_like": "f_standing_tank",
+    "looks_like": "f_air_conditioner",
```

Comment:
this is also what you made the furnace look like, so now these two are going to look identical and usually sit side-by-side. I'd recommend keeping this one, particularly since a water heater looks much more like a tank than an air conditioner.